WalletWasabi.Gui/Controls/WalletExplorer/WalletExplorerViewModel.cs

```diff
@@ -17,32 +19,35 @@ namespace WalletWasabi.Gui.Controls.WalletExplorer
 	[Shared]
 	public class WalletExplorerViewModel : ToolViewModel, IActivatableExtension
 	{
+		private ObservableCollection<WalletViewModelBase> _wallets;
+		private ViewModelBase _selectedItem;
+
 		public override Location DefaultLocation => Location.Right;
 
 		public WalletExplorerViewModel()
 		{
 			Title = "Wallet Explorer";
 
-			_wallets = new ObservableCollection<WalletViewModel>();
+			_wallets = new ObservableCollection<WalletViewModelBase>();
+
+			WalletManager = Locator.Current.GetService<Global>().WalletManager;
 		}
 
-		private ObservableCollection<WalletViewModel> _wallets;
+		private WalletManager WalletManager { get; }
 
-		public ObservableCollection<WalletViewModel> Wallets
+		public ObservableCollection<WalletViewModelBase> Wallets
 		{
 			get => _wallets;
 			set => this.RaiseAndSetIfChanged(ref _wallets, value);
 		}
 
-		private WasabiDocumentTabViewModel _selectedItem;
-
-		public WasabiDocumentTabViewModel SelectedItem
+		public ViewModelBase SelectedItem
 		{
 			get => _selectedItem;
 			set => this.RaiseAndSetIfChanged(ref _selectedItem, value);
 		}
 
-		internal void OpenWallet(Wallet wallet, bool receiveDominant)
+		internal void OpenWallet(Wallet wallet, bool receiveDominant, bool select)
 		{
 			var walletName = Path.GetFileNameWithoutExtension(wallet.KeyManager.FilePath);
 			if (_wallets.Any(x => x.Title == walletName))
```

Comment:
This was not introduced in this PR but it could make sense to change it.

```suggestion
			if (_wallets.Any(x => x.Wallet.Name == walletName))
```

Reply:
I've fixed this expression now.